### DISCUSSION TITLE
第4节课笔记(12班)

### DISCUSSION THREAD

**Discussion by fanqiNO1** [InternLM/Tutorial#243] [笔记]
这里

**Comment by al2h36o30s3** [comment]
https://zhuanlan.zhihu.com/p/677127205?

**Comment by HuangXinzhe** [comment]
https://blog.csdn.net/qq_29787929/article/details/135532991

**Comment by jujimeizuo** [comment]
https://note.jujimeizuo.cn/llm/internlm/lec4/

**Comment by Jeff-Huang-SHU** [comment]
https://www.yuque.com/u34036176/dxc2up/cgc5wvaixsmi9bob?singleDoc# 《Xtuner微调》

**Comment by cheng221** [comment]
https://spice-spice-829.notion.site/1a1dc7130573423eaccd3c58c566e2b0

**Comment by Smiling-Weeping-zhr** [comment]
https://www.cnblogs.com/smiling-weeping-zhr/p/17961091

**Comment by canxingliuying** [comment]
https://blog.csdn.net/weixin_42449385/article/details/135583185

**Comment by ChengaoJ** [comment]
https://blog.csdn.net/weixin_43193528/article/details/135584823?spm=1001.2014.3001.5502

**Comment by ltstar** [comment]
[课程4-xtuner.md](https://github.com/InternLM/tutorial/files/13931537/4-xtuner.md)

**Comment by deseijily** [comment]
https://j1b8l2ps5tm.feishu.cn/docx/DFP3dbNMioQW2axXhtHcPHMsnSb

**Comment by JiangXiaobai00** [comment]
https://d0elb2210mg.feishu.cn/docx/XNxjd33xToSobUx3cJ9cf1q7nHh?from=from_copylink
